source/speech/speech.py

```diff
@@ -2442,6 +2442,17 @@ def getFormatFieldSpeech(  # noqa: C901
 				# Translators: Reported when text is no longer marked
 				else _("not marked"))
 			textList.append(text)
+		# color-highlighted text in Word
+		hlColor = attrs.get("highlight-color")
+		oldHlColor = attrsCache.get("highlight-color") if attrsCache is not None else None
+		if (hlColor or oldHlColor is not None) and hlColor != oldHlColor:

```

Comment:
why is `hlColor` in the first or statement? can `hlColor` be falsey?

Reply:
Like other attributes, it can be None if "highlight-color" key is absent.